code/modules/hydroponics/beekeeping/bee_pack.dm

```diff
@@ -0,0 +1,24 @@
+/obj/item/bee_pack
+	name = "bee pack"
+	desc = "Contains a queen bee and some worker bees. Everything you'll need to start a hive!"
+	icon = 'icons/obj/beekeeping.dmi'
+	icon_state = "beepack"
+	var/full = TRUE
+
+/obj/item/bee_pack/Initialize()
+	. = ..()
+	add_overlay("beepack-full")
+
+/obj/item/bee_pack/proc/empty()
+	name = "empty bee pack"
+	desc = "A stasis pack for moving bees. It's empty."
+	full = FALSE
+	cut_overlays()
+	add_overlay("beepack-empty")
```

Comment:
This stuff could really just be in update_icons() so that you can condense this to 1 proc call on empty and fill.